### DISCUSSION TITLE
Fetching content from (your) other repositories to gh-pages

### DISCUSSION THREAD

**Discussion by kimitobo** [community/community#21938] [Other Feature Feedback, Questions, & Ideas]
<p>I’d like to fetch the README.md files from (some of) my other repositories as the content for pages or posts on my portfolio made with github pages. Can this be automated somehow with Jekyll?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>In a way you can, yes. When you have a GitHub Pages site for your user account at <code>[https://username.github.io](https://username.github.io)</code> you can also turn on GitHub Pages for individual projects that you want connected to that site. When you go into the Settings tab of your project, scroll down to the section marked “GitHub Pages”. Once you’re there you can select a source for the Pages site and choose a theme. When you’ve done that, it will automatically be published to <code>[https://username.github.io/project-name/](https://username.github.io/project-name/)</code></p>
<p>You can find out more about this in the <a href="https://help.github.com/articles/user-organization-and-project-pages/" rel="nofollow noopener">GitHub help on Pages</a>.</p>
